src/components/landing/Landing.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {useState} from 'react';
+import { Link } from 'react-router-dom';
+import './Landing.css';
+import terminal from './landing-images/terminal.gif';
+
+const Landing: React.FC = () => {
+    const [display, setDisplay] = useState('block');
+
+    const startTutorial = () => {
+        setDisplay('none');
+    }
+
+    return (
+        <div className="Landing" style={{display: display}}>
+            <div className="row h-75">
+                <div className="col-xl-6">
+                    <div className="container h-100">
+                        <div className="row align-items-center h-100">
+                            <div className="col-8 mx-auto text-center">
+                                <div className="jumbotron text-center mt-5 mt-xl-0">
```

Comment:
So I see CSS classes like text-center and h-100 for being used in a div and then used again in a nested div right below it. Is that actually required to keep the same styling for the page? I'm not too familiar with bootstrap again, but some of it does feel a little redundant at least from an outside perspective.

Reply:
I've already talked to you in person about this so just resolve this lol. No need to change anything else. Still try to do CSS-in-JS with [styled components](https://www.styled-components.com/) for the custom css you have right now and keep the bootstrap.